static/src/stylesheets/module/commercial/gimbap/_gimbap.scss

```diff
@@ -61,6 +45,87 @@ $gimbap-width-quarter: calc(25% - 20px);
     }
 }
 
+.gimbap--effects {
+    .gimbap__image {
+        transition: all 3.5s ease;
+    }
+
+    &:hover {
+        .gimbap__image {
+            transform: scale(1.25);
+        }
+    }
+}
+
+/* GIMBAP - STYLING */
+.gimbap--tone-members { background: lighten($gimbap-members-text, 36%); color: $gimbap-members-text; }
+.gimbap--tone-masterclasses { background: lighten($gimbap-masterclasses-reversed, 49%); color: $gimbap-masterclasses-text; }
+.gimbap--tone-soulmates { background: lighten($gimbap-soulmates-reversed, 49%); color: $gimbap-soulmates-text; }
+.gimbap--tone-jobs { background: lighten($gimbap-jobs-reversed, 42%); color: $gimbap-jobs-text; }
+.gimbap--tone-bookshop { background: lighten($gimbap-bookshop-reversed, 70%); color: $gimbap-bookshop-reversed; }
+
+.gimbap--normal {
+    .gimbap__cta {
+        filter: brightness(.5);
+    }
+}
+
+.gimbap--reversed {
+    color: #ffffff;
+
+    .gimbap__cta {
+        color: rgba(0, 0, 0, .65);
+    }
+
+    &.gimbap--tone-members { background: $gimbap-members-reversed; }
```

Comment:
In theory yes.

Lets think about the story in which you have a task to change the color tone of `jobs` `reversed` modification. Would you first go to `.gimbap--reversed` and try to change it there or would you go to `.gimbap--tone-jobs` and try to change it there?